reference/3.0/Microsoft.PowerShell.Core/About/about_Execution_Policies.md

```diff
@@ -52,6 +45,19 @@ The PowerShell execution policies are as follows:
   classified as trusted or untrusted.
 - Risks running signed, but malicious, scripts.
 
+### Bypass
+
+- Nothing is blocked and there are no warnings or prompts.
+- This execution policy is designed for configurations in which a PowerShell
+  script is built in to a larger application or for configurations in which
+  PowerShell is the foundation for a program that has its own security model.
+
+### Default
+
+- Sets the default execution policy.
+- **Restricted** for Windows clients
```

Comment:
Add period to list item for consistency.

Reply:
Resolved, thanks for the feedback.